src/gt4py/next/common.py

```diff
@@ -50,6 +50,7 @@
 
 
 DimT = TypeVar("DimT", bound="Dimension")  # , covariant=True)
+DimT_cov = TypeVar("DimT_cov", bound="Dimension", covariant=True)
```

Comment:
The naming convention is `_co` and `_contra`
```suggestion
DimT_co = TypeVar("DimT_co", bound="Dimension", covariant=True)
```